src/index.js

```diff
@@ -26,7 +26,11 @@ export default function factory({
     return nc;
   }
   nc.use = function use(base, ...fns) {
-    if (typeof base === "function") return this.use("/", base, ...fns);
+    if (typeof base === "function") return nc.use("/", base, ...fns);
+    if (Array.isArray(base)) {
+      for (const p of base) nc.use(p, ...fns);
+      return nc;
+    }
```

Comment:
```suggestion
    if (Array.isArray(base)) {
      for (const p of base) nc.use(p, ...fns);
      return nc;
    }
    if (typeof base === "function") return nc.use("/", base, ...fns);
```

More of a personal preference but I would prefer the Array check to be above.